backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,33 +77,93 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
     email.setSubject(subject);
     email.setText(bodyText);
     return email;
   }
 
+  /**
+   * Create a MimeMessage using the parameters provided.
+   *
+   * @param bodyText body text of the email
+   * @return the MimeMessage to be used to send email
+   * @throws MessagingException if there was a problem accessing the Store
+   */
+  private MimeMessage createEmailWithMultiRecipients(
+      String firstRecipientUsername, String secondRecipientUsername, String bodyText)
+      throws MessagingException {
+
+    Properties props = new Properties();
+    Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
+
+    MimeMessage email = new MimeMessage(session);
+
+    email.setFrom(new InternetAddress(APPLICATION_EMAIL));
+    email.addRecipients(
+        RecipientType.TO,
+        new InternetAddress[] {
+          new InternetAddress(firstRecipientUsername + "@google.com"),
+          new InternetAddress(secondRecipientUsername + "@google.com")
+        });
+    email.setSubject("Ad-lib: Match Found");
+    email.setText(bodyText);
+    return email;
+  }
+
   /** Function that access its api and using it sends an email */
-  //   TODO(#35): Create a dummy email for ad lib itself to send emails.
   //   TODO(#36): Replace body to send real link to user instead of generic.
-  public void notifyUser() throws MessagingException, IOException, GeneralSecurityException {
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+  public void sendMatchEmail(String firstMatchRecipientName, String secondMatchRecipientName)
+      throws MessagingException, IOException, GeneralSecurityException {
     MimeMessage email =
-        createEmail(
-            "Ad-Lib Meeting Found",
-            " Hey "
-                + recipientName
+        createEmailWithMultiRecipients(
+            firstMatchRecipientName,
+            secondMatchRecipientName,
+            "Congratulations"
                 + " Please Join your Ad-Lib meeting via the link below : \n"
                 + " http://meet.google.com/new");
     Message messageWithEmail = createMessageWithEmail(email);
     service.users().messages().send("me", messageWithEmail).execute();
   }
+
+  /** Function that access its api and using it sends an email */
+  //   TODO(#36): Replace body to send real link to user instead of generic.
+  public void sendExpiredEmail(String expiredRecipientName) throws MessagingException, IOException {
+    MimeMessage email =
+        createEmailWithSingleRecipient(
+            expiredRecipientName,
+            "Ad-lib: Meeting Query Expired",
+            " We apologize "
+                + expiredRecipientName
+                + "\n"
+                + " You're search for a match took longer than anticipated ! \n"
+                + " Please try again at a later time ! : \n");
```

Comment:
subject: [Ad-lib] Sorry, we couldn't find you a match!
body: 
```Hi recipientUsername,

We couldn't find a match for you this time, but we encourage you to please try again later! 

Best,
The Ad-lib team```

Reply:
Ack